### PR TITLE
Micro Reddit Milestone 1: Associations

### PR DESCRIPTION
**Pull request by jurgen1c**
### Micro Reddit

In this milestone we have:
- Created a User model
- Created a Post model
- Created a Comment model
- Create migration files with table rules
- Tested create,associations and validations of user, post & comment via rails console.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,66 @@
...
-* Deployment instructions
+### Install
+In order to run, you need to install RUBY and Rails in your computer. For windows you can go to [Ruby installer](https://rubyinstaller.org/) and for MAC and LINUX you can go to [Ruby official site](https://www.ruby-lang.org/en/downloads/) for intructions on how to intall it.
 
-* ...
+Then you can clone the project by typing ```https://github.com/jurgen1c/Rails-Micro-Reddit.git```
+
+### Run app
+- Type ```bundle install``` in the root file of the project. 
+- Type ```rails s``` in the root file of the project.
```
Comment (lekegitrepo) [inline README.md]: - **_Include the migration command in the readme instruction._**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes
### Hi @jurgen1c and @RaminMammadzada :clap: :clap:
#### Well done you have done a good job :muscle: However before this project can be approved the project requirements mentioned in my comments below must be implemented.

#### _Check my comments :point_down: and my inline comment below_

* [ ] **_A user can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingUser.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))_**

* [ ] **_A post can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingPost.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))_**

* [ ] **_A comment can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingComment.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))_**

* [ ] **_The tests should also cover the association between the models._**


#### _Reviewed by Temitayo Adeleke._

**Comment by RaminMammadzada** [comment]
> ## Status: Request changes
> ### Hi @jurgen1c and @RaminMammadzada 👏 👏
> #### Well done you have done a good job 💪 However before this project can be approved the project requirements mentioned in my comments below must be implemented.
> #### _Check my comments 👇 and my inline comment below_
> * [ ]  **_A user can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingUser.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))_**
> * [ ]  **_A post can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingPost.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))_**
> * [ ]  **_A comment can be created **(covered by automated tests)** ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/building_with_active_record/creatingComment.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/building-with-active-record-ruby-on-rails))_**
> * [ ]  **_The tests should also cover the association between the models._**
> 
> #### _Reviewed by Temitayo Adeleke._

We followed the requirements of the project. The tests which you asked us to do is beyond the requirement of this project.

File: README.md (hunk 1 above)
Comment (RaminMammadzada) [reply README.md]: Done.

**Review by euqueme — APPROVED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @jurgen1c and @RaminMammadzada :lollipop: :balloon: 

### Thank you for doing the changes requested by the previous reviewer, :1st_place_medal: :1st_place_medal: your project works well and both your Readme and pull request are professional and meet all the mandatory requirements :muscle: is a yes for me so please go ahead and merge this branch Congratulations! :tada: :tada: :tada: 

### Please :pray: Keep up the good work! :clinking_glasses: 

PROJECT STATUS: APPROVED :heavy_check_mark:

**Pull request merged**